### ISSUE TITLE
Possible bug in collapsible panels with ajax

### ISSUE THREAD

**Issue by ScreamingTree86** [TheCoder4eu/BootsFaces-OSP#490]
Hi,
i think there is a bug in the ajax feature of the panels. It occures if you use the `onexpand` and `oncollapse` tags in the Firefox browser. A simple test is to open the you panel showcase site http://showcase.bootsfaces.net/layout/panels.jsf in firefox and chrome. In the ajax and javascript paragraph try to collapse the panel. In chrome it works well but in firefox nothing happens. 

Taking a closer look in the generated js code this is generated: 
```
<script>
    $('#j_idt248_j_idt250content').on('show.bs.collapse', function(e) {
        console.log('expand - start');;
        BsF.ajax.callAjax(this, event, 'j_idt248:messages', '@all', null, null, null, 'expand');
        document.getElementById('j_idt248_j_idt250_collapsed').value = 'false';
    });
</script>
```

As you can see the three JS code lines are called inside a `function(e)` where `e` carrys the event. But `BsF.ajax.callAjax(this, event, 'j_idt248:messages', '@all', null, null, null, 'expand');` use `event`. In chrome there is no Problem, because chrome resolves `event` to `window.event`. But Firefox does not. 
In my test application i get an additional js consol error saying "ReferenceError: event is not defined" clearly pointing at this problem. 

Beside this, i use the `oncollapse` and `onexpand` to persist the collapsed state to a sessionbean. Do you know any workaround to quickfix this and persist the state? 

€dit:
just to be complete. A simple workaround is to use `p:remoteCommand` and let the remotecommand call you backing bean method...

**Comment by stephanrauh** [comment]
@asterd noticed and fixed this bug two weeks ago. At the time, we didn't know it causes problems, so we didn't think too much about this. Would you like to fetch the latest developer snapshot of BootsFaces-1.0.0 and confirm the bug has been fixed? See #369 on how to get it.

Thanks in advance
Stephan

**Comment by ScreamingTree86** [comment]
Okay, i tested it. Works :+1:  

Thank you :)
